test/remote.js

```diff
@@ -29,6 +29,13 @@ const TEST_DATA = [
     remote: 'git@gitlab.com:user/repo.git',
     expected: remotes.gitlab
   },
+  {
+    remote: 'https://gitlab.com/user/repo#subgroup.git',
```

Comment:
I see something like https://gitlab.com/user/repo/subgroup.git in our subgroup project. Are you sure that this is correct form?

Reply:
Real examples are `https://gitlab.rambler.ru/unity/sites/wmj.git` for `https` and `git@gitlab.rambler.ru:unity/sites/wmj.git` for `ssh`

Reply:
Ok good to know. I'll update the tests. I was only using `#` as that is what is used when you `npm install` a certain branch

Reply:
If you need some other urls to test just let me know 😉